sdk/core/core-amqp/test/errors.spec.ts

```diff
@@ -109,7 +109,7 @@ describe("Errors", function() {
         if (
           translatedError.code === "ServerBusyError" ||
           translatedError.code === "MessagingError" ||
-          translatedError.code == undefined
+          translatedError.code === undefined
```

Comment:
This is another one of those places where `==` was intentional

Reply:
Since the tests pass, I don't think adding more ambiguity to this file provides any value, but I'll do the change.